### DISCUSSION TITLE
How do you automatically add a header

### DISCUSSION THREAD

**Discussion by PoIygon** [CluckCluckChicken/Magnifier#77] [Q&A]
Currently working on my website using vscode and I was wondering how to automatically add a header

**Comment by davwat** [comment]
magnifier uses a framework called [blazor ](https://dotnet.microsoft.com/apps/aspnet/web-apps/blazor)where its really easy to implement, but if you dont know c# you should try [nuxt ](https://nuxtjs.org/) or [react](https://reactjs.org/). you can also use an iframe or you can make a script with `document.getElementById("view").appendChild(put your header stuff in here)` or something similar and put that script on every page

**Reply by PoIygon** [reply]
How do you do the document thing

**Reply by davwat** [reply]
make header.js or something with it in then put <script src="/header.js"></script> on every page

**Reply by PoIygon** [reply]
> 
> 
> make header.js or something with it in then put <script src="/header.js"></script> on every page

didnt work
